src/xcode/ENA/ENA/Source/Scenes/Home/HomeTableViewController.swift

```diff
@@ -242,6 +242,7 @@ class HomeTableViewController: UITableViewController, NavigationBarOpacityDelega
 	
 	// swiftlint:disable:next cyclomatic_complexity
 	override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+		tableView.deselectRow(at: indexPath, animated: true)
```

Comment:
since we use a TableViewController here isn't `clearsSelectionOnViewWillAppear` the better option?

Reply:
That does not work in this case